heron/healthm/src/java/com/twitter/heron/healthmgr/common/HealthMgrConstants.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.healthmgr.common;
+
+public interface HealthMgrConstants {
```

Comment:
Fixed.
I have removed the interface. An enum was added where feasible. Guice `Named` qualifier annotation expects a constant String. Such constants cannot be replaced with enum.